File: projects/Python/examples/print_json.py
from proto import test


def main():
    print(test.StructSimple().to_json() + '\n')
    print(test.StructOptional().to_json() + '\n')
    print(test.StructNested().to_json() + '\n')


if __name__ == "__main__":
    main()

File: projects/Swift/Tests/Examples/PrintJson.swift
//
//  PrintJson.swift
//

import XCTest
import ChronoxorTest

class ExamplePrintJson: XCTestCase {
    func testPrintJson() {
        print(try! StructSimple().toJson())
        print("")

        print(try! StructOptional().toJson())
        print("")

        print(try! StructNested().toJson())
        print("")
    }
}
